### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by belozerovany**
@iliatcymbal I did homework 2. Please check my functions

### CODE REVIEW

File: src/compName/compName.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <p><span>Surname:</span> {lastName}</p>
+            <p><span>Age:</span> {age}</p>
+        </div>
+    )
+}
+
+export const ListComponentName = (props) => {
+    const { users } = props;
+    return (
+        <ul className = "ul">
```
Comment (iliatcymbal) [inline src/compName/compName.js]: actually, not very good class name

File: src/compName/compName.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            {
+                users.map((item, index) =>
+                <li className="ul__li" key={index}>
```
Comment (iliatcymbal) [inline src/compName/compName.js]: the same thing - do not use tag names in class names

File: src/content/content.js [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
+                        firstName:'Petia',
+                        lastName :'Petrov',
+                        age :'25' 
+                    },
+                    {
+                        firstName:'Petia',
+                        lastName :'Petrov',
+                        age :'25' 
+                    }
+                ]}
```
Comment (iliatcymbal) [inline src/content/content.js]: it is better do not to use data directly in components. Better way - to crete variable for the users array and pss it to component `users={users}`

File: src/nav/nav.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import'./nav.scss';
+
+export const Nav = (props) => {
+    const navigation = props.list;
+    console.log('navigation',navigation);
```
Comment (iliatcymbal) [inline src/nav/nav.js]: why you have this unnecessary code?

File: src/nav/nav.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    return (
+        <nav className="navigation">
+            <ul>
+                {  
+                    props.list.map((item, iter) =>
```
Comment (iliatcymbal) [inline src/nav/nav.js]: it is good practice to handle case when props.list is undefined

**Review by iliatcymbal — COMMENTED** [review]
Please, fix all notes

**Comment by belozerovany** [comment]
@iliatcymbal i fixed all notes, please check it.

**Review by iliatcymbal — APPROVED** [review]
ok, looks good to me

**Pull request merged**